Public/Sdk/Experimental/NodeJs/npm.dsc

```diff
@@ -41,14 +41,43 @@ namespace Npm {
         };
     }
 
+    @@public
+    export function npmInstall(workingDirectory: StaticDirectory): OpaqueDirectory {

```

Comment:
>workingDirectory [](start = 31, length = 16)

the name looks slightly misleading to me. Given that its type is a static directory, it is already representing a group of artifacts, and not just a working directory. What's the semantics of this collection of files? The output of a compilation?

Reply:
The semantics is:
  - use that folder as the working directory
  - take a dependency on that folder (so that this pip is scheduled after this static directory is produced and that it can read everything in it)

Maybe rename to `rootDir`.